definitions/install.yaml

```diff
@@ -2441,7 +2441,7 @@ spec:
               fieldPath: metadata.namespace
         - name: KYVERNO_SVC
           value: kyverno-svc
-        image: ghcr.io/kyverno/kyverno:v1.3.6-rc1
+        image: ajoop/kyverno:latest
```

Comment:
Looks like you committed your test image by accident.

Reply:
The PR is still in progress. I need to test more cases.